conf/claims.routes

```diff
@@ -8,9 +8,11 @@ GET         /change-number-of-claims                                uk.gov.hmrc.
 POST        /change-number-of-claims                                uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.SelectNumberOfClaimsController.submit()
 
 GET         /enter-movement-reference-number                        uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMovementReferenceNumberController.enterMrn()
```

Comment:
As discussed on call, you were going to have a look at whether it is helpful to make the single journey use the single path prefix at the same time. Would like to discuss whatever solution you come up with.